lib/dependenciesInstall.js

```diff
@@ -37,34 +37,23 @@ module.exports = function(context) {
         return Q();
     }
 
-    var dependencies = packageJSON.dependencies;
-
-    var depNames = [];
-    for (var dep in dependencies) {
-        if (dependencies.hasOwnProperty(dep)) {
-            depNames.push(dep);
-        }
-    }
-
-    var packagesToInstall = depNames.map(function(dep) {
-        return dep + '@' + dependencies[dep];
-    });
-
     var npm = context.requireCordovaModule('npm');
     return Q.promise(function(resolve, reject, notify) {
         npm.load({
             loaded: false
         }, function(err) {
             if (err) {
-                console.log('cordova-plugin-livereload: An error occurred while loading npm');
-                reject('cordova-plugin-livereload: npm load failed.');
+                console.log(pluginName + ': An error occurred while loading npm');
```

Comment:
Why do we chose to log here and not higher level? Also, we seem to ignore the `errr` itself.